Source/ASTextNode2.mm

```diff
@@ -321,18 +324,31 @@ - (NSArray *)exclusionPaths
   return _textContainer.exclusionPaths;
 }
 
-- (void)prepareAttributedString:(NSMutableAttributedString *)attributedString
+- (void)prepareAttributedString:(NSMutableAttributedString *)attributedString forIntrinsicSize:(BOOL)isForIntrinsicSize
```

Comment:
This method name implies the second parameter is a `CGSize`, to me something like `isForIntrinsicSize` would imply a `BOOL`.